### ISSUE TITLE
no bottleneck, freeze encoder

### ISSUE THREAD

**Issue by yesWeGAN** [yesWeGAN/buddE#8]
## 12.12.2023
- trained 224-image-size model on COCO using bottleneck dim of 512 (was 256), converged after roughly 50 epochs, giving performance increase especially in small / medium boxes. Maybe those small features are lost in the bottleneck
- next step: do not use a bottleneck, freeze pretrained encoder layers
